### ISSUE TITLE
Screen regions not recognized on (some?) multi-monitor setups

### ISSUE THREAD

**Issue by skewty** [emasab/shelltile#86]
gnome 3.32 (ubuntu 19.04)

My arrangement is as follows:
[1920x1200 (portrait right, centered)] [2560x1600 (landscape, main)]

The extension does not recognize the top-left and bottom-left regions on my main (right) monitor. 
All regions are recognized on the secondary (left) monitor.

Is it because no corners exist on the left side of my main / right monitor?

Here is metadata.json from the version I am running (from extensions.gnome.org):
```javascript
{
  "_generated": "Generated by SweetTooth, do not edit", 
  "description": "A tiling window extension for GNOME Shell. Just move a window to the edges of the screen to create a tiling, otherwise move a window over another one, holding down the Control key. Grouped windows minimize, resize, raise and change workspace together. Move or maximize a window to remove it from the group.", 
  "gettext-domain": "shelltile", 
  "name": "ShellTile", 
  "settings-schema": "org.gnome.shell.extensions.shelltile", 
  "shell-version": [
    "3.6", 
    "3.8", 
    "3.10", 
    "3.12", 
    "3.14", 
    "3.16", 
    "3.18", 
    "3.20", 
    "3.22"
  ], 
  "url": "https://github.com/emasab/shelltile", 
  "uuid": "ShellTile@emasab.it", 
  "version": 59
}
```

**Comment by SimonBo** [comment]
I have similar issue on my 3-monitor setup, and it's also the rightmost monitor's top and bottom left regions that are not recogninzed.

**Comment by SimonBo** [comment]
Well, it actually works correctly after disabling Grouping edge tiling! @skewty maybe this will solve your issue too.

**Comment by skewty** [comment]
Yeah, disabling "Group edge tiling" seems to make it work on my Gnome Shell 3.34 installation :)  

Thanks!

**Comment by emasab** [comment]
@SimonBo Hi Simon, your rightmost monitor is in portrait mode like @skewty 's?

**Comment by SimonBo** [comment]
@emasab It's landscape

**Comment by emasab** [comment]
@SimonBo is your rightmost monitor the main one? Are you using Dash to Panel too?

**Comment by igordovgaluk** [comment]
the extension breaks tiling with mouse on the common edge of dual monitor setup. After the extension is disabled the default tiling on the screen edge works better.

**Comment by emasab** [comment]
@igordovgaluk take into consideration that the default active region is one pixel in size but can be increased in the settings with the "Size of active edge tiling zone" setting.

**Comment by dmusican** [comment]
@igordovgaluk Increasing size of active edge tiling zone solved the problem for me, thank you. It seems that this isn't so much a bug rather than a default setting that needs improvement, and possibly some documentation.
